chaincfg/params.go

```diff
@@ -270,6 +275,11 @@ var MainNetParams = Params{
 			StartTime:  1199145601, // January 1, 2008 UTC
 			ExpireTime: 1230767999, // December 31, 2008 UTC
 		},
+		DeploymentCSV: {
+			BitNumber:  0,
+			StartTime:  1462060800, // March 1st, 2016
```

Comment:
Yep, I noticed that a bit ago and fixed it. Looks like this comment was made from an outdated diff.